src/components/layout/error-reporting-notice/error-reporting-notice.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { ERROR_REPORTING_CONSENT_KEY_NAME, images, isErrorReportingAllowed } from '../../../utils';
+import Button from '../../button/button';
+import Link from '../../link';
+import styles from './error-reporting-notice.module.scss';
+
+interface WelcomeModalContentProps {
+  onClose: () => void;
+}
+
+const ErrorReportingNotice = (props: WelcomeModalContentProps) => {
```

Comment:
This needs to be fixed to the bottom (floating above content), instead of below it. You can't see it on the staking page without scrolling down

Reply:
Let's discuss this one in slack.